.github/workflows/R-CMD-check.yaml

```diff
@@ -34,9 +34,6 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@v2
       
-      - name: Install curl
```

Comment:
This step threw an error during CI, couldn't find the library. Removed it, apparently it's not needed (CI for PKPDsim also doesn't install this).